modin/pandas/base.py

```diff
@@ -1480,25 +1480,35 @@ def iloc(self):
         return _iLocIndexer(self)
 
     def kurt(self, axis=None, skipna=None, level=None, numeric_only=None, **kwargs):
-        return self._default_to_pandas(
-            "kurt",
-            axis=axis,
-            skipna=skipna,
-            level=level,
-            numeric_only=numeric_only,
-            **kwargs
-        )
+        """Return unbiased kurtosis over requested axis. Normalized by N-1
 
-    def kurtosis(self, axis=None, skipna=None, level=None, numeric_only=None, **kwargs):
-        return self._default_to_pandas(
-            "kurtosis",
-            axis=axis,
-            skipna=skipna,
-            level=level,
-            numeric_only=numeric_only,
-            **kwargs
+        Kurtosis obtained using Fisher’s definition of kurtosis (kurtosis of normal == 0.0).
+
+        Args:
+            axis : {index (0), columns (1)}
+            skipna : boolean, default True
+                Exclude NA/null values when computing the result.
+            level : int or level name, default None
+            numeric_only : boolean, default None
+
+        Returns:
+            kurtosis : Series or DataFrame (if level specified)
+        """
+        axis = self._get_axis_number(axis)
+        if numeric_only is not None and not numeric_only:
```

Comment:
There may be a logic bug here

```python
if numeric_only is not None and numeric_only:
```

We are validating the dtypes with the `numeric_only` flag set to `True`. Was this logic pulled from elsewhere because it looks like a bug, or it should be documented.

Reply:
I took from here(some examples):
- https://github.com/modin-project/modin/blob/6e2df02fd6bc17cbed27246f20b5975bd1032ab5/modin/pandas/base.py#L1641
- https://github.com/modin-project/modin/blob/6e2df02fd6bc17cbed27246f20b5975bd1032ab5/modin/pandas/base.py#L2559

Reply:
I see, this is a bug. pandas attempts to convert non-numeric types to float, and if that fails it will throw a different error than we do. Fixing that bug will block this PR, then the casting utility can be used by this.

```python
df = pd.DataFrame({"A":[1,2,3,4], "B":list("NDCS")})
df.median(numeric_only=False)
df = pd.DataFrame({"A":[1,2,3,4], "B":list("1234")})
df.median(numeric_only=False)
```

Reply:
Changed to `if numeric_only is not None and numeric_only:`